api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -449,3 +470,30 @@ message Plan {
   // successfully and data processing is complete.
   repeated TelemetryMetadata telmetry_metadata = 13;
 }
+
+// Request for the 'AddTle' method.
+message AddTleRequest {
+  // The ID of the satellite to add TLE for. The ID can be found on the StellarStation
+  // Console page for the satellite.
+  string satellite_id = 1;
+
+  // The TLE to be added.
+  orbit.Tle tle = 2;
+}
+
+// Response for the `AddTle` method.
+message AddTleResponse {
+  // Currently no payload in the response.
+}
+
+// Request for the 'GetTle' method.
+message GetTleRequest {
+  // The ID of the satellite for which to fetch TLE. The ID can be found on the StellarStation
+  // Console page for the satellite.
+  string satellite_id = 1;
+}
+
+// Response for the 'GetTle' method.
+message GetTleResponse {
+  orbit.Tle tle = 1;
```

Comment:
`// The latest TLE for the given satellite.`

Reply:
Please be more specific about definition of `latest`. latest epoch?

Reply:
We could also drop 'latest' entirely (especially since the 'add' API is becoming 'set').
// The TLE for the given satellite.

What is being returned here is the 'active' TLE (which happens to be the one with the most recent epoch that stellarstation knows about).

Reply:
I dropped latest. The rpc comment mentions that the tle is the 'current' tle which I believe would be understood to mean 'the TLE used by stellarstation when computing the orbit for this satellite'.